ui/src/components/FileEditor/Editor.vue

```diff
@@ -48,9 +48,9 @@ const theme = computed(() => {
   return 'xcode'
 })
 
-const selectedFileName = computed(() => store.state.fileEditorModule.selectedFileName)
-const isHelpOpen = computed(() => store.state.fileEditorModule.isHelpOpen)
-const fileString = computed(() => store.state.fileEditorModule.file)
+const selectedFileName = computed(() => fileEditorStore.selectedFileName)
+const isHelpOpen = computed(() => fileEditorStore.isHelpOpen)
```

Comment:
One other shortcut is that we should be able use these right in the template without computing them. Like `<div :class="{ 'help-open': fileEditorStore.isHelpOpen...` But again, this can be optimized after.

Reply:
Good idea, I'll try to apply both of these suggestions in the future and eventually clean up existing code to apply them.